blockchain/state/iterator.go

```diff
@@ -184,7 +187,7 @@ func (it *NodeIterator) retrieve() bool {
 }
 
 // CheckStateConsistency checks the consistency of all state/storage trie of given two state database.
-func CheckStateConsistency(oldDB database.DBManager, newDB database.DBManager, root common.Hash) error {
+func CheckStateConsistency(oldDB database.DBManager, newDB database.DBManager, root common.Hash, mapSize int, quit chan struct{}) error {
```

Comment:
Do you think it is unnecessary to check `accountHash` and `codeHash`?

Reply:
Yes it is.
Account trie node includes codeHash. And AccountHash is the position of a trie node.
So I think we need to check them.

Reply:
Okay. They are retrieved on `Next()` and checked on `oldIt.Hash != newIt.Hash`. Thanks.